### PR TITLE
Fix ellipsis overflow causing edit icon to be hidden on links

### LINKED ISSUE #5064: Ellipsis doesn't take into account clickable fields
We have an issue with long texts:
Screenshot 2024-04-19 at 16 23 38
Screenshot 2024-04-19 at 16 25 22

As you can see, for the address or input fields that are not clickable it works well.
But for input fields that have a pen icon (e.g. link, email...) then it becomes impossible to modify the input if you input something too long.
Instead, the ellipsis should come earlier and we should see the pen icon

Marking this as good first issue but it might not be that easy - tbc!

### PR DESCRIPTION
**Pull request by DevanandGowda**
Fixes #5064 

### Demo

https://github.com/twentyhq/twenty/assets/21654351/28ab7374-c57e-4f7e-9720-05138c53a33d

### CODE REVIEW

**Comment by FelixMalfait** [comment]
Thanks a lot

That's the easy way to solve it.
For pills it doesn't bother me to have a max-width everywhere.
But for the email field for example it's not great as it has an impact on list views:
Screenshot 2024-04-20 at 11 09 36

I wonder if you can find something smarter to avoid hardcoding a max width.
Ideally w we should adapt when there's a buttonIcon or find another solution that doesn't affect list views.
It might be a difficult problem but it'd be great if you can try - we can probably do better!

**Comment by DevanandGowda** [comment]
Ahh good catch, wasn't aware the columns are extendable, so this effects all the fields including the email field. 

I will see if I can try to avoid hardcoding the width while accomodating the edit button, might be possible by using something like `visibility: hidden` to retain the space occupied by the edit button.

**Comment by DevanandGowda** [comment]
@FelixMalfait I went with the above idea, mainly becuase I felt it's more consistent with what's present in the table view. Additionally this also fixes the issue you caught wrt email. Here's the demo, let me know what you think. Thanks. 


https://github.com/twentyhq/twenty/assets/21654351/7e997937-7601-448f-91d2-b7bcb2d2e891

**Review by FelixMalfait — APPROVED** [review]
Thanks!

File: packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCellContainer.tsx [hunk 1]
```diff
@@ -39,6 +39,7 @@ const StyledLabelAndIconContainer = styled.div`
 
 const StyledValueContainer = styled.div`
   display: flex;
+  min-width: 0;
```
[the remark this answers is not in the document]
Comment (DevanandGowda) [reply packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCellContainer.tsx:42]: By default flex items have `min-width: auto` which was the cause for this bug. You can read more about it here in this [amazing SO answer](https://stackoverflow.com/a/36247448)

[the remark this answers is not in the document]
Comment (FelixMalfait) [reply packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCellContainer.tsx:42]: Interesting! Thanks!!!

**Pull request merged**